pkg/process/checks/process.go

```diff
@@ -146,6 +140,15 @@ func (p *ProcessCheck) Init(syscfg *SysProbeConfig, info *HostInfo) error {
 	p.disallowList = initDisallowList(p.config)
 
 	p.initConnRates()
+
+	if workloadmeta.Enabled(p.config) {
+		err = p.workloadMetaServer.Start()
+		if err != nil {
+			_ = log.Error("Failed to start the workload meta gRPC server:", err)
```

Comment:
Let's be more descriptive around the server which this is trying to bring up, which to my understanding is a process entity server.